keyboards/ducky/one2sf/1967st/config.h

```diff
@@ -20,3 +21,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define DIP_SWITCH_MATRIX_GRID { {0,14}, {1,14}, {2,14}, {3,14} }
 
 #define GPIO_INPUT_PIN_DELAY (NUC123_HCLK / 6 / 1000000L)
+
+#define RGB_MATRIX_LED_COUNT 69
```

Comment:
This isn't needed.  It will be automatically computed based on the info.json config.

Reply:
Done, thanks.